modules/claims_api/config/initializers/okcomputer.rb

```diff
@@ -97,4 +97,4 @@ def name
 OkComputer::Registry.register 'bgs-contention', BgsCheck.new('contention')
 OkComputer::Registry.register 'vbms', VbmsCheck.new
 
-OkComputer.make_optional %w[vbms bgs-vet_record bgs-claimant bgs-contention]
+OkComputer.make_optional %w[vbms bgs-vet_record bgs-contention]
```

Comment:
bgs-claimant is actually used for detecting poa if the api consumer is a Veteran representative